css/properties.json

```diff
@@ -120,6 +120,21 @@
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
+  "-moz-context-properties": {
+    "syntax": "none | [ fill | stroke ]#",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "Mozilla Extensions"
+    ],
+    "initial": "none",
+    "appliesto": "replacedElements",
```

Comment:
It doesn't just apply to replaced elements. It applies to any element that can reference an image.